getting-started/frames.ipynb

```diff
@@ -1107,69 +1094,1600 @@
+       "      <th>11</th>\n",
+       "      <th>windows</th>\n",
+       "      <td>1.412696</td>\n",
+       "      <td>20.436635</td>\n",
+       "      <td>29.898394</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-12-04 23:31:18.260</th>\n",
+       "      <th>11</th>\n",
+       "      <th>windows</th>\n",
+       "      <td>92.302583</td>\n",
+       "      <td>37.375834</td>\n",
+       "      <td>2.454959</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-12-05 06:49:33.913</th>\n",
+       "      <th>11</th>\n",
+       "      <th>windows</th>\n",
+       "      <td>31.443326</td>\n",
+       "      <td>58.476529</td>\n",
+       "      <td>86.797440</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-12-05 14:07:49.565</th>\n",
+       "      <th>11</th>\n",
+       "      <th>windows</th>\n",
+       "      <td>19.366588</td>\n",
+       "      <td>69.632766</td>\n",
+       "      <td>98.109378</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-12-05 21:26:05.217</th>\n",
+       "      <th>11</th>\n",
+       "      <th>windows</th>\n",
+       "      <td>4.499338</td>\n",
+       "      <td>75.110088</td>\n",
+       "      <td>90.717712</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2019-12-06 04:44:20.869</th>\n",
+       "      <th>11</th>\n",
+       "      <th>windows</th>\n",
+       "      <td>83.516672</td>\n",
+       "      <td>23.327160</td>\n",
+       "      <td>93.248763</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                            mem        cpu       disk\n",
+       "time                    node os                                      \n",
+       "2019-12-03 11:00:00.000 11   windows  48.638564  72.253130  88.366517\n",
+       "2019-12-03 18:18:15.652 11   windows  27.742501  87.686276  61.147908\n",
+       "2019-12-04 01:36:31.304 11   windows  94.197867  86.861995  79.770651\n",
+       "2019-12-04 08:54:46.956 11   windows  97.307368  96.169328   3.146355\n",
+       "2019-12-04 16:13:02.608 11   windows   1.412696  20.436635  29.898394\n",
+       "2019-12-04 23:31:18.260 11   windows  92.302583  37.375834   2.454959\n",
+       "2019-12-05 06:49:33.913 11   windows  31.443326  58.476529  86.797440\n",
+       "2019-12-05 14:07:49.565 11   windows  19.366588  69.632766  98.109378\n",
+       "2019-12-05 21:26:05.217 11   windows   4.499338  75.110088  90.717712\n",
+       "2019-12-06 04:44:20.869 11   windows  83.516672  23.327160  93.248763"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Read the full table contents, as in the previous example but use an SQL query\n",
+    "query_str = os.path.join(\"select * from '\" + tsdb_table + \"'\")\n",
```

Comment:
`query_str = "select * from ' + tsdb_table + ')`

Reply:
@talIguaz changed in commit b9378e8, which also makes other related cosmetics updates (`os.path.join` — receive path variables; use `f` for formatting with variables in more locations; reduce empty lines at end of cells). I did a full-execution test of all cells and everything worked well.